lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -243,6 +245,7 @@ def self.upload_binary(app_id, binary_path)
         end
       end
 
+      # When the binary is uploaded response body includes the release id and so that is stored for future upload methods.
```

Comment:
I think this would be better if it described the exact shape of the returned data. If you want to include the reason why, that's optional. But if you're going to include a doc string for the method, a quick summary of what it does is usually good to have first. So, for example:
```
# <what it does, e.g. "Uploads a binary...">
#
# <describe the return value, e.g. "Returns a tuple of..."><reason, e.g. "because we use the release ID to...">
```